icssc-projects/peterportal-client
# Dockerize

Dockerize app

## Description
Setup docker-compose and Dockerfile. Added local mongo/elasticsearch instances. 

## Steps to verify/test this change:
1. Install Docker
2. Run docker-compose up on project root
3. Open localhost:3000 on browser and check if app runs

## Todos:
- [ ] Get hot reload to work
